data/json/itemgroups/clothing.json

```diff
@@ -188,5 +188,90 @@
       [ "pocketwatch", 25 ],
       [ "wristwatch", 100 ]
     ]
+  },
+  {
+    "id": "clothing_technician",
```

Comment:
The only usage is in `mon_zombie_technician_death_drops` so you can safely combine them into `clothing_work_set` and then move the technician zombie to it's own file under `monsterdrops`